app.ts

```diff
@@ -0,0 +1,163 @@
+﻿/* Grid object class
+ *   this contains the entire grid object and methods to retrieve data from server 
+ *   and parse data to html strings for displaying in grid table on page
+ */
+class Grid {
+	element: HTMLElement;
+	nr_of_records: number;
```

Comment:
Variables should be camelCase. See https://imqssoftware.atlassian.net/wiki/display/AR/JavaScript+Style+Guide